pgxn/neon/relkind_cache.c

```diff
@@ -0,0 +1,334 @@
+/*-------------------------------------------------------------------------
+ *
+ * relkind_cache.c
+ *      Cache for marking unlogged relations
```

Comment:
```suggestion
 *      Cache to track the relkind of relations
```
it's not just for unlogged relations